debugger/debugger.c

```diff
@@ -37,22 +38,24 @@ bool exec_cmd (Emulator *emu, char *line, int len)
   //printf("Got %s, %u, %u - ops %d\n", cmd, op1, op2, ops);
 
   /* RESET / RESTART
-     
+
      Resets the entire virtual machine to it's starting state.
      Puts the starting address back into Program Counter
    */
   if ( !strncasecmp("reset", cmd, sizeof "reset") ||
```

Comment:
Why not `strcasecmp(const char *s1, const char *s2);` ?

Reply:
all `n` functions are considered "safer" and their "n less" equivalents are often marked as unsafe by various static analyzers

buuuut in this case indeed it's unnecessary

Reply:
NOP (also, not our code)